configure.ac

```diff
@@ -1620,6 +1620,7 @@ ac_libs_save="$LIBS"
 LIBS="$LIBS $LIBYANG_LIBS"
 AC_CHECK_FUNC([ly_register_types], [
   libyang_ext_builtin=true
+  ac_libs_save="$LIBS"
```

Comment:
I have no idea what exactly you're trying to fix here but this is incorrect and doesn't affect LIBYANG_LIBS.

Reply:
you should try the following:
`LIBYANG_CFLAGS="..." LIBYANG_LIBS = "..." ./configure ..`

in that case the LIBS variable is not saved, and we go back to the old value.
which means that you can not compile frr with libyang if you parameterize LIBYANG_XXX variables.

For my puropose, I commented out ac_libs_save. but for general frr purpose, it is not good. So I try to find a way to not ignore the LIBYANG_XXX flags.

Reply:
```
$ LIBYANG_LIBS="-lyang -Ltest_ignore_this" ./configure [...]
[...]
$ grep \"LIBYANG_LIBS config.status 
S["LIBYANG_LIBS"]="-lyang -Ltest_ignore_this"
```

It's working correctly for me...